h2o-r/h2o-package/R/connection.R

```diff
@@ -763,3 +763,14 @@ h2o.networkTest <- function() {
 .h2o.garbageCollect <- function() {
   res <- .h2o.__remoteSend("GarbageCollect", method = "POST")
 }
+
+#' Open H2O Flow
+#'
+#' Open H2O Flow in your browser
+#'
+#' @param ip Object of class \code{character} representing the IP address of the server where H2O is running.
+#' @param port Object of class \code{numeric} representing the port number of the H2O server.
+#' @export
+h2o.flow <- function(ip = "localhost", port=54321){
```

Comment:
The h2o connection does have the required information, there is no need to specify these arguments by the user

We also need to respect the URL context (when Steam is used)